recipes/libde265/run_test.sh

```diff
@@ -0,0 +1,5 @@
+git clone https://github.com/strukturag/libde265-data.git
```

Comment:
In addtion to testing that your binaries can load. Please use [existence tests](https://conda-forge.org/docs/maintainer/adding_pkgs.html#simple-existence-tests) to explicitly check that the following were installed to the correct place:

- all libs (both .lib and .dll on Windows)
- one of the cmake files
- one of the headers

These check will prevent accidental removals or ABI breaking name changes from going un-noticed in the future.